pages/item/_.vue

```diff
@@ -274,7 +274,7 @@
         const noSimilarItems = { results: [] };
         if (this.error) return noSimilarItems;
 
-        if (this.$auth.loggedIn) {
+        if (this.$auth.loggedIn && Boolean(Number(process.env.ENABLE_RECOMMENDATIONS))) {
```

Comment:
I think we may benefit ever so slightly from switching these around as I'd imagine the loggedIn check is a bit more expensive since it needs to check the token and might result in a redirect to log in again if the token has expired.

Reply:
Yes, good suggestion!